File: cours_Info/algo/_static/codes/chap3/convertTemperatureCtoK.f90
!convertTemperatureCtoK
!----------------------
!Print temperature in Kelvin from input read temperature in Celcius
PROGRAM convertTemperatureCtoK
  IMPLICIT NONE
  REAL, PARAMETER :: zeroC = 273.15d0
  REAL :: tempC, tempK
  PRINT*, "Enter temperature in Celcius :"
  READ*, tempC
  tempK = tempC + zeroC
  PRINT*, tempC, " °C is ", tempK, " K"
END PROGRAM convertTemperatureCtoK

File: cours_Info/algo/_static/codes/chap3/convertTemperatureCtoK.c
//convertTemperatureCtoK
//----------------------
//Print temperature in Kelvin from input read temperature in Celcius
#include <stdio.h>
main()
{
  const float zeroC(273.15);
  float tempC,tempK;
  printf("Enter temperature in Celcius :\n");
  scanf(&tempC);
  tempK = tempC + zeroC;
  printf("%f °C is %f K\n", tempC, tempK);
}